js/ui/map.js

```diff
@@ -57,47 +57,61 @@ var defaultOptions = {
 };
 
 /**
- * Creates a map instance. This is usually the beginning of your map:
- * you tell Mapbox GL JS where to put the map by specifying a `container`
- * option, and the map's style with `style` and other attributes of the map,
- * and in return Mapbox GL JS initializes the map on your page and returns
- * a map variable that lets you programmatically call methods on the map.
+ * A [`LngLat`](#LngLat) object or an array of two numbers representing longitude and latitude.
+ * @typedef {(LngLat | Array<number>)} LngLatLike
+ * /
+
+/**
+ * A [`LngLatBounds`](#LngLatBounds) object or an array of [`LngLatLike`](#LngLatLike) objects.
+ * @typedef {(LngLatBounds | Array<LngLatLike>)} LngLatBoundsLike
```

Comment:
Added these `@typedef`s, but not sure how to get them to render in a good place via documention.js. Seems like maybe they aren't being parsed correctly, either ... or my syntax is off. @tmcw Any insight here?